Userland/Applications/Browser/Tab.cpp

```diff
@@ -144,6 +144,16 @@ Tab::Tab(BrowserWindow& window, Type type)
     m_location_box = toolbar.add<GUI::TextBox>();
     m_location_box->set_placeholder("Address");
 
+    m_location_box->on_control_return_pressed = [this] {
+        StringBuilder pre_url;
+        pre_url.append("www.");
+        pre_url.append(m_location_box->text());
+        pre_url.append(".com");
```

Comment:
Firefox only autocompletes URLs when there isn't already a dot inside, and automatically adds the `.com` before the first slashes which seems like a good heuristic. You could probably implement something similar here :)

Some examples fo Firefox's autocompletes:
- `test` => `www.test.com`
- `test.com` => `test.com`
- `www.test` => `www.test` (I think it's not adding the .com in case `test` is a valid TLD)
- `test/thing` => `www.test.com/thing`